tools/test_containers_compose

```diff
@@ -5,8 +5,8 @@ set -euo pipefail
 for i in webui worker; do
   (
     cd container/$i/ &&
-    sudo docker-compose build --parallel &&
+    (for retry in 2 1 0; do sudo docker-compose build && exit 0; echo "Remaining reties $retry"; done; exit 1) &&
```

Comment:
s/reties/retries/

Also how about:

```suggestion
    for retry in {2..0}; do sudo docker-compose build && break; echo "Remaining retries $retry"; done &&
```

You should not need the subshell with just "break"

Reply:
Done